test/rpc/validation.js

```diff
@@ -0,0 +1,59 @@
+const test = require('tape')
+const request = require('supertest')
+
+const { startRPC, closeRPC } = require('./helpers')
+const { middleware } = require('../../lib/rpc/validation')
+
+test('should comlete the request with non-params if the method does not require any params', t => {
+  const server = startRPC({
+    'mock': middleware((params, cb) => {
+      cb()
+    }, 0, [])
+  })
+  const req = {
+    jsonrpc: '2.0',
+    method: 'mock',
+    id: 1
+  }
+
+  request(server)
+    .post('/')
+    .set('Content-Type', 'application/json')
+    .send(req)
+    .expect(res => {
```

Comment:
We additionally need a success case for this - like e.g. [here](https://github.com/ethereumjs/ethereumjs-client/blob/master/test/rpc/eth/getBlockByHash.js#L39) - this should likely be `expect(200)` when looking at the code.

Reply:
Awesome! I'll merge and add tests.